### PR TITLE
Documentation improvements

### PR DESCRIPTION
**Pull request by peelar**
## Context
We received feedback from @aniav about the difficulties in making the first steps with apps. This PR is one of the attempts to improve it.

## Features
- Linguistic fixes to the `README.md`.
- Added some extra content to the template's introductory page. It now includes instructions on installing the app in the dashboard, links to Next.js documentation, and the "tunneling the apps" article. Some small fixes with it as well.

## Screenshot

image

### CODE REVIEW

**Review by aniav — COMMENTED** [review]
👌

**Review by krzysztofwolski — APPROVED** [review]

File: src/pages/index.tsx [hunk 1]
```diff
@@ -62,16 +66,65 @@ const IndexPage: NextPage = () => {
...
+            App Manifest
+          </a>
+          .
+        </li>
+        <li>
+          <code>src/pages/api/webhooks/order-created</code> - an example <code>ORDER_CREATED</code>{" "}
+          webhook handler.
+        </li>
+        <li>
+          <code>graphql</code> - the pre-defined GraphQL queries.
```
Comment (lkostrowski) [inline src/pages/index.tsx:94]: maybe lets mark it more as a directory. `/graphql` or a simple icon. Other paths the same

File: src/pages/index.tsx [hunk 2]
```diff
@@ -138,8 +193,48 @@ const IndexPage: NextPage = () => {
...
+                Installation
+              </Text>
+              <Text as={"p"} variant="body" marginTop={2}>
+                To install the app in your Saleor Dashboard, follow these steps:
+              </Text>
+              <ol>
+                <li>
+                  1. Copy the URL of your{" "}
+                  <a
+                    href="https://docs.saleor.io/docs/3.x/developer/extending/apps/developing-with-tunnels"
```
Comment (lkostrowski) [inline src/pages/index.tsx:208]: we have now 2 ways of doing it. Maybe lets give here 2 "paths" - with tunnels and locally. Just link two article to chose from

File: src/pages/index.tsx [hunk 3]
```diff
@@ -138,8 +193,48 @@ const IndexPage: NextPage = () => {
...
         <>
-          <Text marginBottom={4} as={"p"}>Install this app in your Dashboard and check extra powers!</Text>
-          {mounted && !global.location.href.includes("localhost") && <AddToSaleorForm />}
+          {isLocalHost && (
+            <Box>
+              <Text variant={"heading"} marginTop={8} as={"h2"}>
+                Installation
+              </Text>
+              <Text as={"p"} variant="body" marginTop={2}>
+                To install the app in your Saleor Dashboard, follow these steps:
```
Comment (lkostrowski) [inline src/pages/index.tsx:202]: It doesnt fit well if below we have this `<form>`. I mean its a different of doing the same field.

So if we give manual steps of installation, ensure its `.... OR ....`

File: src/pages/index.tsx (hunk 2 above)
Comment (lkostrowski) [reply src/pages/index.tsx:208]: Im also not sure if these steps are required.

The form below is basically doing the same thing but automatically, under the hood. 

IMO its just complicating things.

Learn about tunneling or local development - needed
Explain steps on dashboard - not needed

**Review by lkostrowski — CHANGES_REQUESTED** [review]

**Comment by peelar** [comment]
@lkostrowski addressed

**Review by lkostrowski — APPROVED** [review]

**Pull request merged**